resources/ext/build-scripts/install-vendored-gems.sh

```diff
@@ -15,9 +15,11 @@ cat "${DIR}/gem-list.txt"
 
 echo "jruby-puppet: { gem-home: ${DESTDIR}/opt/puppetlabs/server/data/puppetserver/vendored-jruby-gems }" > jruby.conf
 
+gem_list=()
 while read LINE
 do
   gem_name=$(echo $LINE |awk '{print $1}')
   gem_version=$(echo $LINE |awk '{print $2}')
-  java -cp puppet-server-release.jar:jruby-1_7.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install ${gem_name} --version ${gem_version}
+  gem_list+=("$gem_name:$gem_version")
 done < "${DIR}/gem-list.txt"
+java -cp puppet-server-release.jar:jruby-1_7.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install --no-ri --no-rdoc "${gem_list[@]}"
```

Comment:
Not part of your PR, but worth noting that the sem_pup version is totally both different than what is in puppet-agent/master and different than what is vendored in puppet itself. (I don't think you need to change anything, but I think someone should look at it).